Error creating DB indexes on server startup

**Steps to reproduce**

Error log on startup
```
     Reason: liquibase.exception.DatabaseException: Duplicate key name 'IDX_PORTAL_SITES_TYPE_01' [Failed SQL: (1061) CREATE INDEX IDX_PORTAL_SITES_TYPE_01 ON exo.PORTAL_SITES(TYPE)] [o.e.c.p.impl.LiquibaseDataInitializer<main>] 
liquibase.exception.LiquibaseException: liquibase.exception.MigrationFailedException: Migration failed for change set db/changelog/portal-rdbms.db.changelog-1.0.0.xml::1.0.0-37::portal:
     Reason: liquibase.exception.DatabaseException: Duplicate key name 'IDX_PORTAL_SITES_TYPE_01' [Failed SQL: (1061) CREATE INDEX IDX_PORTAL_SITES_TYPE_01 ON exo.PORTAL_SITES(TYPE)]
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:126)
	at liquibase.Liquibase.lambda$null$0(Liquibase.java:265)
	at liquibase.Scope.lambda$child$0(Scope.java:180)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:158)
	at liquibase.Scope.child(Scope.java:243)
	at liquibase.Liquibase.lambda$update$1(Liquibase.java:264)
	at liquibase.Scope.lambda$child$0(Scope.java:180)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:158)
	at liquibase.Liquibase.runInScope(Liquibase.java:2405)
	at liquibase.Liquibase.update(Liquibase.java:211)
	at liquibase.Liquibase.update(Liquibase.java:197)
	at liquibase.Liquibase.update(Liquibase.java:193)
	at liquibase.Liquibase.update(Liquibase.java:185)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.applyChangeLog(LiquibaseDataInitializer.java:169)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.initData(LiquibaseDataInitializer.java:150)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.initData(LiquibaseDataInitializer.java:112)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.start(LiquibaseDataInitializer.java:99)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.exoplatform.container.LifecycleVisitor.traverse(LifecycleVisitor.java:119)
	at org.exoplatform.container.LifecycleVisitor.start(LifecycleVisitor.java:185)
	at org.exoplatform.container.ConcurrentContainer.start(ConcurrentContainer.java:472)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:67)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:67)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:67)
	at org.exoplatform.container.ExoContainer.start(ExoContainer.java:309)
	at org.exoplatform.container.PortalContainer.start(PortalContainer.java:638)
	at org.exoplatform.container.ExoContainer.start(ExoContainer.java:282)
	at org.exoplatform.container.PortalContainer.start(PortalContainer.java:645)
	at org.exoplatform.container.ExoContainer.start(ExoContainer.java:282)
	at org.exoplatform.container.RootContainer.createPortalContainer(RootContainer.java:781)
	at org.exoplatform.container.RootContainer.createPortalContainers(RootContainer.java:360)
	at org.exoplatform.platform.server.tomcat.PortalContainersCreator$1.run(PortalContainersCreator.java:50)
	at org.exoplatform.platform.server.tomcat.PortalContainersCreator$1.run(PortalContainersCreator.java:48)
	at org.exoplatform.commons.utils.SecurityHelper.doPrivilegedAction(SecurityHelper.java:291)
	at org.exoplatform.platform.server.tomcat.PortalContainersCreator.createPortalContainers(PortalContainersCreator.java:48)
	at org.exoplatform.platform.server.tomcat.PortalContainersCreator.lifecycleEvent(PortalContainersCreator.java:37)
	at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:114)
	at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:402)
	at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:345)
	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:893)
	at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:795)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1332)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1322)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
	at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:145)
	at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:866)
	at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:249)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171)
	at org.apache.catalina.core.StandardService.startInternal(StandardService.java:433)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171)
	at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:922)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171)
	at org.apache.catalina.startup.Catalina.start(Catalina.java:772)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:347)
	at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:478)
Caused by: liquibase.exception.MigrationFailedException: Migration failed for change set db/changelog/portal-rdbms.db.changelog-1.0.0.xml::1.0.0-37::portal:
     Reason: liquibase.exception.DatabaseException: Duplicate key name 'IDX_PORTAL_SITES_TYPE_01' [Failed SQL: (1061) CREATE INDEX IDX_PORTAL_SITES_TYPE_01 ON exo.PORTAL_SITES(TYPE)]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:696)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:56)
	at liquibase.changelog.ChangeLogIterator$2.lambda$null$0(ChangeLogIterator.java:113)
	at liquibase.Scope.lambda$child$0(Scope.java:180)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:158)
	at liquibase.changelog.ChangeLogIterator$2.lambda$run$1(ChangeLogIterator.java:112)
	at liquibase.Scope.lambda$child$0(Scope.java:180)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:158)
	at liquibase.Scope.child(Scope.java:243)
	at liquibase.changelog.ChangeLogIterator$2.run(ChangeLogIterator.java:93)
	at liquibase.Scope.lambda$child$0(Scope.java:180)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:158)
	at liquibase.Scope.child(Scope.java:243)
	at liquibase.Scope.child(Scope.java:247)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:65)
	... 67 common frames omitted
Caused by: liquibase.exception.DatabaseException: Duplicate key name 'IDX_PORTAL_SITES_TYPE_01' [Failed SQL: (1061) CREATE INDEX IDX_PORTAL_SITES_TYPE_01 ON exo.PORTAL_SITES(TYPE)]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:397)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:83)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:151)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1279)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1261)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:661)
	... 87 common frames omitted
Caused by: java.sql.SQLSyntaxErrorException: Duplicate key name 'IDX_PORTAL_SITES_TYPE_01'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:118)
	at jdk.proxy3/jdk.proxy3.$Proxy12.execute(Unknown Source)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:393)
	... 92 common frames omitted
```

--- a/component/portal/src/main/resources/db/changelog/portal-rdbms.db.changelog-1.0.0.xml
+++ b/component/portal/src/main/resources/db/changelog/portal-rdbms.db.changelog-1.0.0.xml
@@ -386,6 +386,13 @@
 
   <!-- Add index for performance -->
   <changeSet author="portal" id="1.0.0-37">
+    <preConditions onFail="MARK_RAN">
+      <not><indexExists indexName="IDX_PORTAL_SITES_TYPE_01" tableName="PORTAL_SITES"/></not>
+      <not><indexExists indexName="IDX_PORTAL_SITES_NAME_01" tableName="PORTAL_SITES"/></not>
+      <not><indexExists indexName="IDX_PORTAL_LOCALE_01" tableName="PORTAL_SITES"/></not>
+      <not><indexExists indexName="IDX_PORTAL_SKIN_01" tableName="PORTAL_SITES"/></not>
+      <not><indexExists indexName="IDX_PORTAL_LABEL_01" tableName="PORTAL_SITES"/></not>
+    </preConditions>
     <createIndex indexName="IDX_PORTAL_SITES_TYPE_01"
                  tableName="PORTAL_SITES">
       <column name="TYPE" type="INT"/>
@@ -406,18 +413,10 @@
                  tableName="PORTAL_SITES">
       <column name="LABEL" type="NVARCHAR(200)"/>
     </createIndex>
-    <createIndex indexName="IDX_PORTAL_DESCRIPTION_01"
-                 tableName="PORTAL_SITES">
-      <column name="DESCRIPTION" type="LONGTEXT"/>
-    </createIndex>
-    <createIndex indexName="IDX_PORTAL_PROPERTIES_01"
-                 tableName="PORTAL_SITES">
-      <column name="PROPERTIES" type="LONGTEXT"/>
-    </createIndex>
-    <createIndex indexName="IDX_PORTAL_SITE_BODY_01"
-                 tableName="PORTAL_SITES">
-      <column name="SITE_BODY" type="LONGTEXT"/>
-    </createIndex>
+  </changeSet>
+
+  <!-- Add index for performance -->
+  <changeSet author="portal" id="1.0.0-38">
     <createIndex indexName="IDX_PORTAL_SITES_DISPLAYED_01"
                  tableName="PORTAL_SITES">
       <column name="DISPLAYED" type="BOOLEAN"/>

